src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  font-family: $font-family;
+  padding: 16px;
+  transition: transform 1s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin: 16px 0 0;
+  }
+
+  &__header {
+    color: $color-black;
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    margin: 40px 0 0;
+
+  }
+
+  &:hover &__header {
+    color: #34568b;
+  }
+
+  &__product_code {
+    font-weight: 400;
+    font-size: 10px;
+    margin: 4px 0 0;
+    line-height: 14px;
+    color: $grey-color;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 0 0;
+    line-height: 14px;
+
+    &-counter {
+      display: flex;
+      font-family: $font-family;
+      font-weight: 400;
+      font-size: 10px;
+      margin: 0;
+      align-items: center;
+    }
+  }
+
+  &__price {
+    line-height: 18px;
+    display: flex;
+    justify-content: space-between;
+    margin: 24px 0 0;
+
+    &-text {
+      text-align: center;
+      color: $grey-color;
+      font-size: 12px;
+      font-weight: 400;
+      margin: 0;
+    }
+
+    &-value {
+      color: $color-black;
+      font-weight: 700;
+      margin: 0;
+    }
+  }
+
+  &__button {
+    font-family: $font-family;
+    box-sizing: border-box;
+    height: 40px;
+    margin: 16px 0 0;
```

Comment:
As with the other margin properties, use the shorthand notation for margin here to comply with the code style rules.